client/apps/account-settings/components/label-payment-method/index.js

```diff
@@ -32,18 +32,18 @@ const PaymentMethod = ( { translate, selected, type, digits, name, expiry, onSel
 	};
 
 	return (
-		<CompactCard className="shipping__card" onClick={ onSelect }>
+		<CompactCard className="label-payment-method" onClick={ onSelect }>
```

Comment:
should this file be renamed just `.../payment-method/index.js`? The component name is `<PaymentMethod />`.